src/pages/resultsView/pathwayMapper/PathWayMapperContainer.tsx

```diff
@@ -10,13 +10,98 @@ import ResultsViewURLWrapper from 'pages/resultsView/ResultsViewURLWrapper';
 import { AppStore } from 'AppStore';
 import { useLocalObservable } from 'mobx-react-lite';
 import { runInAction } from 'mobx';
+import request from 'superagent';
+import { remoteData } from 'cbioportal-frontend-commons';
 
 interface IPathwayMapperContainerProps {
     resultsViewPageStore: ResultsViewPageStore;
     appStore: AppStore;
     urlWrapper: ResultsViewURLWrapper;
 }
 
+interface ResultItem {
+    databases: DatabaseItem[];
+}
+interface DatabaseItem {
+    name: string;
+    numberOfNetworks: string;
+    url: string;
+}
+interface ApiResponse {
+    results: ResultItem[];
+}
+
+function makeRemoteData() {
+    return remoteData<DatabaseItem[]>({
+        await: () => [],
+        invoke: async () => {
+            const getResponse = await request
+                .get(
+                    'https://iquery-cbio-dev.ucsd.edu/integratedsearch/v1/source'
+                )
+                .set('Accept', 'application/json')
+                .timeout(60000)
+                .redirects(0);
+
+            const jsonData: ApiResponse = getResponse.body;
+            const extractedData: DatabaseItem[] = jsonData.results[0]?.databases.map(
+                result => ({
+                    name: result.name,
+                    numberOfNetworks: parseInt(
+                        result.numberOfNetworks,
+                        10
+                    ).toString(),
+                    url: result.url,
+                })
+            );
+            return extractedData;
+        },
+        default: [],
+    });
+}
+
+const TooltipContent: React.FC = observer(() => {
+    const store = useLocalObservable(() => ({
+        tooltipContent: null as DatabaseItem[] | null,
+        tooltipData: makeRemoteData(),
+    }));
+
+    return (
+        <div style={{ maxWidth: 400 }}>
+            <a href="https://www.ndexbio.org/" target="_blank">
+                NDEx
+            </a>{' '}
+            shows 1,352 pathways:
+            {store.tooltipData.isPending ? (
+                <div>Loading tooltip content...</div>
+            ) : (
+                <div>
+                    {store.tooltipData.result && (
+                        <ul>
+                            {store.tooltipData.result.map(
+                                (item: DatabaseItem, index: number) => (
+                                    <li key={index}>
+                                        {item.numberOfNetworks} from{' '}
+                                        <a
+                                            href={`https://${item.url}`}
```

Comment:
@coleslaw481 mentioned:

> You can just use the value of the url field. The value may lack a https:// prefix so that should be checked for and added if missing

It looks like in the code here we assume `item.url` never contains the protocol? Maybe better to be safe and strip out all protocol stuff from `item.url` (i.e. `http{s}://`) if it exists?